src/views/docs/en/guides/developer-experience/using-esm.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using ESM (ECMAScript modules)
```

Comment:
Style tweak: I've been using the formal name: `ES Modules` (or `ECMAScript Modules`), cap `M`